app/consumers/compliance_reports_consumer.rb

```diff
@@ -60,7 +60,9 @@ def validate
   end
 
   def validation_message
-    XccdfReportParser.new(@report_contents, @msg_value)
+    @report_contents.each do |report|
+      XccdfReportParser.new(report, @msg_value)
+    end
```

Comment:
Should we allow partial processing of an archive if one of the reports fails? The StandardError rescue below will halt parsing for all remaining reports in the archive if that error is raised for any one report.